server/src/rest_service.py

```diff
@@ -1,7 +1,37 @@
 from config import config
 import requests
+from schema import User
+
 
 def slack_users():
     token_data = {'token': config['slackToken']}
     return requests.get(config['slackApiUrl'] + '/users.list', params=token_data)
 
+
+def app_users(session):
+    return session.query(User).all()
+
+
+# probably works? have not tested
```

Comment:
Another nitpick that doesn't matter much at all: Feels strange to have methods for both HTTP requests and DB queries in the same file. We can maybe think about refactoring later.